language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -14,7 +14,6 @@ import qualified Data.Text.Extended as T
 import qualified "zip-archive" Codec.Archive.Zip as Zip
 import qualified Data.Map as Map
 
-import Control.Monad
```

Comment:
I’m confused, how did this compile before? Was this not a redundant import?

Reply:
We have `Control.Monad.Extra` and I introduced `Control.Monad` sort of accidentally in a prior commit. I'm fixing that. No, it didn't get flagged as redundant.

Reply:
Hm, interesting. Not sure why GHC did not detect this.